en/bio.html

```diff
@@ -35,25 +35,14 @@
 }(document, 'script', 'facebook-jssdk'));</script>
   <div id="main">
     <div id="top_margin"></div>
-  <div id="flags" style="float:right"> 
-    <ul>
-      <a href="/cat"><img src="/img/cat.jpg" height="15" width="15" /></a>
-      <a href="/es"><img src="/img/es.jpg" height="15" width="15" /></a>
-      <a href="/en"><img src="/img/en.jpg" height="15" width="15" /></a>
-      <a href="/da"><img src="/img/da.jpg" height="15" width="15" /></a>
-    </ul>
-  </div>
     <header>
       <div id="logo">
         <a href="/en"><h1>CESAR<span id="surname">JOANIQUET</span></h1></a></div>
       <nav>
         <ul class="lava" id="lava_menu">
-          <li><a href="/en/news.html">News</a></li>
           <li class="current"><a href="/en/bio.html">Bio</a></li>
-      
           <li><a href="/en/gallery.html">Gallery</a></li>
           <li><a href="/en/discography.html">Discography</a></li>
-          <li><a href="/en/store.html">Store</a></li>

```

Comment:
we want to make the page more minimalist, simple and focussed. We remove store and news sections